cacheops/simple.py

```diff
@@ -46,8 +48,20 @@ def wrapper(*args, **kwargs):
                 cache_key = 'c:' + _get_key(func, args, kwargs, extra)
                 try:
                     result = self.get(cache_key)
+
                 except CacheMiss:
-                    result = func(*args, **kwargs)
+                    if USE_LOCK:
+                        # This will block in all calls but the
+                        # first to avoid dog-pile effect.
+                        with redis_lock.Lock(self.conn, cache_key):
```

Comment:
On code terseness, it's really a matter of habit and a bit of taste. More
empty lines - every code fragment is a bit more clear, less - overall
picture is more evident.

Anyway, thanks very much for all your effort.